### DISCUSSION TITLE
[Lychee 5.3.0] How to config exiftool...

### DISCUSSION THREAD

**Discussion by vuongquoc** [LycheeOrg/Lychee#2439] [Q&A]
Hi,
I'm a noobie on coding and i'm stuck with decoding GPS info so i though i could give ExifTool enable a try...
So i've installed ExifTool in /perl/usr/bin
but when i enable it in settings, i can not upload pictures anymore...
Error Logs create 2 new errors:

----
128:/home/whcallo6t34w/public_html/bachda.xyz/app/Jobs/ProcessImageJob.php Given path () to the exiftool binary is invalid
Context:

[
  {
    "file": "...../vendor/lychee-org/php-exif/lib/PHPExif/Adapter/Exiftool.php",
    "line": 101,
    "function": "setToolPath",
    "class": "PHPExif\\Adapter\\Exiftool",
    "type": "->"
  },
 ...
And alot more after that
----

and this

----
PHPExif\Adapter\Exiftool::setToolPath:48 Given path () to the exiftool binary is invalid
App\Metadata\Extractor::createFromFile:98 Given path () to the exiftool binary is invalid; caused by
----

So, what and where should i config/change/edit to enable the exiftool?

I've tried to edit the path in Exiftool.php

----
protected string $toolPath = '/home/*****/perl/usr/bin/';
----

and the above 2 errors is no longer happen when i upload new picture and i can upload picture but another error is 

----
PHPExif\Adapter\Exiftool::getExifFromFile:154 Could not decode exiftool output
App\Metadata\Extractor::createFromFile:98 Could not decode exiftool output; caused by
----

So i think there must be configured somewhere...

Please help me if you have any spare time... 

Thank you.

VuongQuoc

**Comment by d7415** [comment]
I don't suppose you have SSH access? If so, try running your version of exiftool manually and see what the output looks like.

**Reply by vuongquoc** [reply]
Thank you, @d7415 
I have SSH access, but i don't know what command to run exiftool... i'm just a noobie in this coding...

Vuong Quoc
